9-regular-expressions/17-regexp-methods/article.md

```diff
@@ -95,13 +95,19 @@ Splits the string using the regexp (or a substring) as a delimiter.
 We can use `split` with strings, like this:
 
 ```js run
-alert('12-34-56'.split('-')) // array of [12, 34, 56]
+alert('12-34-56'.split('-')) // array of ['12', '34', '56']
 ```
 
 But we can split by a regular expression, the same way:
 
 ```js run
-alert('12, 34, 56'.split(/,\s*/)) // array of [12, 34, 56]
+alert('12, 34, 56'.split(/,\s*/)) // array of ['12', '34', '56']
+```
+
+Also we can use `limit` for limit output, like this:
```

Comment:
Maybe: 'for limited output' or 'to limit output'.

Reply:
@vsemozhetbyt Thanks, I'll fix it